content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+Also general performance improvements have been integrated that affect all platforms. Kudos to [RoosterDragon](https://github.com/RoosterDragon) for his continuous effort to optimize our source code.
+
+The refined system integration brings several new polish:
+
+  - The hardware cursors will now always work on Microsoft Windows.
+  - On Mac the dialogs won't hijack finder and the dock icon won't get lost upon restarting. Minimum supported OS X version is now 10.6.
+  - Linux AppStream compatible software centers will now show a nice preview.
+  - We updated our [HTML server browser](/games/) with filters and clickable URLs for your convenience.
+
+Our new in-game map editor took almost a year of development time and it was worth it:
+
+  - It is truely cross-platform as it uses our game UI so finally a map editor on Mac OS X!
+  - No more rendering glitches. Almost WYSIWYG editing, because we re-use the game renderer.
+  - Support for all file formats, i.e. voxels, isometric terrain and Dune 2000 sprite actors.
+  - You can now manage installed custom maps conveniently.
+
+Great news for tinkerers. You can now use Voxels in your own mods and maps. See our updated [trait documentation](https://github.com/OpenRA/OpenRA/wiki/Traits-(playtest)).
+The [Lua API](https://github.com/OpenRA/OpenRA/wiki/Lua-API-(playtest)) for mission scripting has also been enhanced while we proceed to implement remakes of the original campaign.
+
+More details and links to pull request can be found on our [changelog page](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)).
+
+You can find the installers for a variety of supported operating systems in our [download section](/download/).
+
+Please report problems [on our bug tracker](http://bugs.openra.net). We plan to do a new stable release upon these changes soon.
+
+<div style="text-align:center" markdown="1">
+
+![Red Alert map chooser](/images/news/20150524-ra-map-chooser.png)
+
+Keep track of your thirdparty maps from [OpenRA Resource Center](http://resource.openra.net/) and delete unwanted ones in the updated map chooser.
+
+![Tiberian Dawn mission Nod06a](/images/news/20150524-cnc-nod06a.png)
+
+One by one the results of the student projects is integrated. Big thanks to the course instructors at [THM](http://www.thm.de/site/en/) for allowing such an awesome way to teach real-life software development.
+
+![Dune 2000 building upgrades](/images/news/20150524-d2k-building-upgrades.png)
+
+The on-going quest to [finalize the Dune 2000](https://github.com/OpenRA/OpenRA/issues/7751) mod has reached a new milestone with building upgrades which re-create the original tech tree.
+
+![Tiberian Sun in-game map editor](/images/news/20150524-ts-ingame-editor.png)
+
+The in-game map editor is also a vital part of our [Tiberian Sun support](https://github.com/OpenRA/OpenRA/issues/5350) initiative. Click `Extras` - `Map Editor` from the main menu.
```

Comment:
+ "in any mod." ? Just so that people don't go "I want to see that editor but I don't have a TS mod".